spec/01-unit/003-prefix_handler_spec.lua

```diff
@@ -181,22 +181,26 @@ describe("NGINX conf compiler", function()
         assert.not_matches("proxy_ssl_certificate_key.*spec/fixtures/kong_spec.key", kong_nginx_conf)
       end)
     end)
-    it("does not include lua_ssl_trusted_certificate/lua_ssl_verify_depth by default", function()
+    it("sets lua_ssl_verify_depth", function()
+      local conf = assert(conf_loader(helpers.test_conf_path, {
+        lua_ssl_verify_depth = "2"
+      }))
+      local kong_nginx_conf = prefix_handler.compile_kong_conf(conf)
+      assert.matches("lua_ssl_verify_depth 2;", kong_nginx_conf, nil, true)
+    end)
```

Comment:
Will you also add a test that asserts that `lua_ssl_verify_deps is set by default`? Thanks.